internal/pkg/source/entry.go

```diff
@@ -79,8 +81,11 @@ func parseField(parsedLine any, field config.Field) string {
 		}
 
 		unquotedField, err := strconv.Unquote(string(jsonField))
-		if err != nil {
-			return string(jsonField)
+		// It's possible that what were were given is an integer or float
+		// in which case, calling Unquote isn't doing us a lot of good.
+		// Therefore, we just convert to a string value and proceed.
+		if err == strconv.ErrSyntax {
```

Comment:
```suggestion
		if err != nil {
```

in case of any error, we still need to simply use `jsonField`